torch/onnx/symbolic_opset13.py

```diff
@@ -2,12 +2,15 @@
 # see Note [Edit Symbolic Files] in symbolic_helper.py
 
 # This file exports ONNX ops for opset 13
+from sys import maxsize
 from torch.onnx.symbolic_helper import _block_list_in_opset
 import torch
 import torch.onnx.symbolic_helper as sym_help
 from torch.onnx.symbolic_helper import parse_args
+from torch.onnx.symbolic_opset9 import overload_by_arg_count, _maybe_cast_reduce_op_input
+from torch.onnx.utils import _add_block, _add_input_to_block, _add_output_to_block
 
-block_listed_operators = ['embedding_bag']
+block_listed_operators = ['']
```

Comment:
Thanks. If block_listed_operators is empty and not used, please remove this and lines 15-16.

Reply:
Thanks, removed it. And also update the code in reducesum symbolics to use the helper function.